packages/cloudflare/src/client.ts

```diff
@@ -29,8 +29,13 @@ export class CloudflareClient extends ServerRuntimeClient<CloudflareClientOption
   }
 }
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-interface BaseCloudflareOptions {}
+interface BaseCloudflareOptions {
+  /**
+   * @ignore Only used internally to signal that the SDK is being used in a workflow.
+   * @hidden Only used internally to signal that the SDK is being used in a workflow.
+   */
+  isWorkflow?: boolean;
```

Comment:
I would prefer if we rename this to `enableDedupe?: boolean` and then just say it's enabled for `instrumentWorkflowWithSentry`

Reply:
`enableDedupe` with a default of `true` and we set to false for workflows?

Reply:
yes - sorry I think I incorrectly reversed it during my review.